lib/kafka_ex.ex

```diff
@@ -299,7 +299,14 @@ defmodule KafkaEx do
     offset_commit_api_version = Keyword.get(opts, :offset_commit_api_version, 0)
 
     retrieved_offset =
-      current_offset(supplied_offset, partition, topic, worker_name)
+      current_offset(
+        supplied_offset,
+        partition,
+        topic,
+        worker_name,
+        offset_commit_api_version,
```

Comment:
Nit-pick on this: I'm wondering if we shouldn't offer a choice to the user whether they wish to use `offset_commit_api_version` also for `offset_fetch_api_version`.

In any case, I'm thinking that maybe some explicit variable name would help the reader follow the logic and make it explicit that we've decided that `offset_fetch_api_version = offset_commit_api_version`.

Suggestion:

```elixir
    offset_commit_api_version = Keyword.get(opts, :offset_commit_api_version, 0)
    # By default, it makes sense to synchronize the API version of the offset_commit and the offset_fetch
    # operations, otherwise we might commit the offsets in zookeeper and read them from Kafka, meaning
    # that the value would be incorrect.    
    offset_fetch_api_version = Keyword.get(opts, :offset_fetch_api_version, offset_commit_api_version)

    retrieved_offset =
      current_offset(
        supplied_offset,
        partition,
        topic,
        worker_name,
        offset_fetch_api_version,
```

Reply:
It makes real sense, I pushed the correction :)